pkg/apis/provisioning/v1alpha1/provisioner.go

```diff
@@ -106,14 +106,22 @@ var (
 	// Reserved labels
 	ProvisionerNameLabelKey      = SchemeGroupVersion.Group + "/name"
 	ProvisionerNamespaceLabelKey = SchemeGroupVersion.Group + "/namespace"
-	ProvisionerUnderutilizedKey  = SchemeGroupVersion.Group + "/underutilized"
-	ProvisionerTTLKey            = SchemeGroupVersion.Group + "/ttl"
+	ProvisionerPhaseLabel        = SchemeGroupVersion.Group + "/lifecycle-phase"
+
+	// Reserved annotations
+	ProvisionerTTLKey = SchemeGroupVersion.Group + "/ttl"
 
 	// Use ProvisionerSpec instead
 	ZoneLabelKey         = "topology.kubernetes.io/zone"
 	InstanceTypeLabelKey = "node.kubernetes.io/instance-type"
 )
 
+const (
+	ProvisionerUnderutilizedPhase = "underutilized"
```

Comment:
Do you mind capitalizing these? Better k8s style I think. Check out other resources like Pod and Node

Reply:
I think I'd rather keep it lowercase here. Since these are labels, I want to keep it in line with how the other labels are done, which they're all lowercase.